roles/bwc/tasks/ldap.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- name: Setup st2.conf auth backend to LDAP
+  become: yes
+  ini_file:
+    dest: "{{ st2_config_file_path }}"
+    section: auth
+    option: backend
+    value: ldap
+    backup: yes
```

Comment:
Include notify for every task where its needed logically (ex: changing the conf file), its less error prone since "next task" could be removed, adjusted or never ran because of some conditional.

It doesn't matter how many times we call Ansible handlers, - they will be executed in the end of entire play run in any way.

See "Handlers: Running Operations On Change": http://docs.ansible.com/ansible/playbooks_intro.html with better explanation how it works.